sklearn/linear_model/tests/test_logistic.py

```diff
@@ -541,6 +541,46 @@ def test_logistic_regressioncv_class_weights():
     assert_array_almost_equal(clf_lib.coef_, clf_lbf.coef_, decimal=4)
 
 
+def test_logistic_regressioncv_sample_weights():
```

Comment:
Hmm. But if someone makes the code paths independent in the future, we should be make sure that it is tested.
You can just add one test or write models = [LogisticRegression(), LogisticRegressionCV()] and loop over models. (Just to be on the safer side)